tcms/xmlrpc/api/testplan.py

```diff
@@ -7,7 +7,7 @@
 from tcms.testplans.models import TestPlan
 from tcms.testcases.models import TestCase, TestCasePlan
 
-from tcms.xmlrpc.forms import EditPlanForm, NewPlanForm
+from tcms.xmlrpc.forms import RPCEditPlanForm, RPCNewPlanForm
```

Comment:
This should be something like `tcms.xmlrpc.api.forms.testplan import EditPlanForm, NewPlanForm`.

If you re-organize where the classes are defined and make the modules smaller, then you can keep the existing class names (resulting in smaller diff below) because now you know where the form classes are coming from. If the RPC app is self contained (but dependent on other base classes) then we don't really need to overload all class names with meaning, like it is now, and later have to change the names because the meaning has changed.

Reply:
Ok, but again I will have the predefined name problem, e.g.
`xmlrpc.api.forms.testplan`:
```python
from tcms.testplans.forms import NewPlanForm

class NewTestPlan(NewTestPlan):
    pass

class EditTestPlan(NewTestPlan):
    pass
```

IMO this change can be left for later, because we need to do the same for the TestCase forms - the mess there is similar to the one we are fixing now.

We can then come up with package names and class names for all the forms

Reply:
I have done that reorganization. IMO the next steps here should be to do the same for TestCase RPC forms - the mess there is almost the same as the one we are fixing now

Reply:
btw for this I had to squash the two commits:
- Delete XMLRPCEditPlanForm and XMLRPCNewPlanForm
- Introduce RPCPlanForm as base form

because otherwise we had the problem with duplicate names and imports you have exposed above.